control-plane/cmd/kafka-source-controller/main.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"log"
+
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/signals"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection/sharedmain"
+
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/config"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/consumer"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/consumergroup"
+	sourcev2 "knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/source/v2"
+)
+
+const (
+	component = "kafka-source-controller"
+)
+
+func main() {
+
+	consumerEnv, err := config.GetEnvConfig("SOURCE")
+	if err != nil {
+		log.Fatal("cannot process environment variables with prefix SOURCE", err)
+	}
+
+	sharedmain.MainNamed(signals.NewContext(), component,
+
+		// KafkaSource controller
+		injection.NamedControllerConstructor{
+			Name: "source-controller",
+			ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+				return sourcev2.NewController(ctx)
+			},
+		},
+
+		// ConsumerGroup controller
+		injection.NamedControllerConstructor{
+			Name: "consumergroup-controller",
+			ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+				return consumergroup.NewController(ctx)
+			},
+		},
+
+		// Consumer controller
+		injection.NamedControllerConstructor{
+			Name: "consumer-controller",
+			ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
+				return consumer.NewController(ctx, consumerEnv)
```

Comment:
In my PR, I removed `consumerEnv` since we don't need all the variables defined in consumerEnv (https://github.com/knative-sandbox/eventing-kafka-broker/commit/e97c79fa7d982f744a1af3ee1ee1054c8eccb49a), can we do the same here? (maybe in a separate PR?)

Reply:
Yes, will do in this PR. Thanks